packages/mux-player/src/template.ts

```diff
@@ -78,7 +82,6 @@ export const content = (props: MuxTemplateProps) => html`
       player-software-name="${props.playerSoftwareName ?? false}"
       player-software-version="${props.playerSoftwareVersion ?? false}"
       env-key="${props.envKey ?? false}"
-      stream-type="${props.streamType ?? false}"
       custom-domain="${props.customDomain ?? false}"
       src="${!!props.src
```

Comment:
still need to solve `cast-stream-type` below

Reply:
Resolved by adding support to `<castable-video>` and migrating to the "real" version 🎉!